docs/reference/detr.md

```diff
@@ -0,0 +1,395 @@
+## detr module
+
+The *detr* module contains the *DetrLearner* class, which inherits from the abstract class *Learner*.
+
+### Class DetrLearner
+Bases: `engine.learners.Learner`
+
+The *DetrLearner* class is a wrapper of the DETR [[1]](#detr-paper) object detection algorithm based on the original [DETR implementation](https://github.com/facebookresearch/detr).
+It can be used to perform object detection on images (inference) and train DETR object detection models.
+
+The [DetrLearner](#src.perception.object_detection_2d.detr.detr_learner.py) class has the
+following public methods:
+
+#### `DetrLearner` constructor
+```python
+DetrLearner(model_config_path, iters, lr, batch_size, optimizer, backbone, checkpoint_after_iter, checkpoint_load_iter, temp_path, device, threshold, num_classes, return_segmentations)
```

Comment:
Could you please add `self` to all the methods to be consistent with the rest of the documentation?
```suggestion
DetrLearner(self, model_config_path, iters, lr, batch_size, optimizer, backbone, checkpoint_after_iter, checkpoint_load_iter, temp_path, device, threshold, num_classes, return_segmentations)
```

Reply:
It is still missing in the constructor.

Reply:
I am sorry about that, it should be added now.